cedar-testing/src/cedar_test_impl.rs

```diff
@@ -394,6 +408,21 @@ impl CedarTestImplementation for RustEngine {
         };
         TestResult::Success(response)
     }
+    fn validate_entities(
+            &self,
+            schema: &ValidatorSchema,
+            entities: Entities,
+        ) -> TestResult<TestValidationResult> {
+        panic!("TODO");
```

Comment:
Let's fill these in before merging the PR. Should be similar to the code you added in the `cedar-spec` PR.